03-solution-demos/streaming-game-analytics/README.md

```diff
@@ -0,0 +1,233 @@
+# Build Real-Time Player Analytics  with RisingWave
+Create a streaming table in RisingWave that ingests data from a Kafka topic, run windowed KPIs, and surface leaders & spikes — all in SQL, all in real time.
```

Comment:
please provide a bit more details here, and use a tone that's more approachable.